jitterdebugger.c

```diff
@@ -464,6 +460,10 @@ static void start_measuring(struct stats *s, struct record_data *rec)
 					"Check your affinity mask\n");
 			err_handler(err, "pthread_create()");
 		}
+
+		err = pthread_setaffinity_np(s[i].pid, sizeof(mask), &mask);
```

Comment:
In this case I would suggest to move the `pthread_setaffinity_np` call into the worker, otherwise there is the race that the worker is migrated after the measurement has started.